### PR TITLE
Master issue 12 comments

### PR DESCRIPTION
**Pull request by IraVolyk**
#12 

1. Added migration and validation for comments.
2. Added create, edit and destroy functionality.
3. Added UI for comments. 
image

4. Added infinite pagination and scrolling the card with "perfect-scrollbar".

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      end
+    end
+  end
+
+  def edit; end
+
+  def update
+    @comment.update(comment_params)
+    respond_to do |f|
+      if @comment.valid?
```
Comment (yuravenzhik) [inline app/controllers/comments_controller.rb:32]: errors.empty?

File: app/controllers/comments_controller.rb [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+      else
+        f.js { flash[:error] = @comment.errors.full_messages.join("\n") }
+      end
+    end
+  end
+
+  def edit; end
+
+  def update
+    @comment.update(comment_params)
```
Comment (yuravenzhik) [inline app/controllers/comments_controller.rb:30]: лишняя строка

Comment (IraVolyk) [reply app/controllers/comments_controller.rb]: убрали

File: app/controllers/comments_controller.rb (hunk 1 above)
Comment (IraVolyk) [reply app/controllers/comments_controller.rb]: Сделали if @comment.update(comment_params)

**Pull request merged**